mapengine/worldwind/aar/build.gradle

```diff
@@ -3,6 +3,7 @@ apply from: "$rootDir/build-support/emp3-gradle-plugin/src/main/resources/androi
 dependencies {
     compile          ("com.android.support:support-annotations")
     provided project (":emp3-android-sdk")
+    provided files ("$rootDir/3rdpartylibrary/cmapi/cmapi.jar")
```

Comment:
I thought about it.  The version is 0.0.7.   We're still waiting for something like a 1.0